metaflow/plugins/argo/argo_workflows.py

```diff
@@ -800,6 +802,12 @@ def _container_templates(self):
             # support Metaflow sandboxes
             env["METAFLOW_INIT_SCRIPT"] = KUBERNETES_SANDBOX_INIT_SCRIPT
 
+            # secrets stuff
+            env["METAFLOW_DEFAULT_SECRETS_BACKEND_TYPE"] = DEFAULT_SECRETS_BACKEND_TYPE
+            env[
+                "METAFLOW_AWS_SECRETS_MANAGER_DEFAULT_REGION"
+            ] = AWS_SECRETS_MANAGER_DEFAULT_REGION
```

Comment:
We would need a similar change for Airflow too

Reply:
Done!  @valayDave  would you be able to lend a hand in testing this on Airflow?  It should be a case of just setting a single @secrets and running this on oleg2.

Reply:
Changes work with Airflow.